resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/config.scala

```diff
@@ -152,6 +152,20 @@ package object config extends Logging {
       .stringConf
       .createOptional
 
+  private[spark] val KUBERNETES_DRIVER_SECRETS =
+    ConfigBuilder("spark.kubernetes.driver.secrets")
+      .doc("Secrets to be mounted into the driver Pod. This should be a comma-separated list of" +
```

Comment:
We should probably use the newer format of key-value pair configurations. See what we do for annotations and labels.

Reply:
Done.